docs/README.md

```diff
@@ -0,0 +1,258 @@
+# Axiom V1 Core Contracts
```

Comment:
Is it intentional that there's README.md and docs/README.md?

Reply:
No created both in case we wanted to have more extensive contributor docs.
For now, I have removed the README.md in docs.
I have also removed the examples from README.md and only provided links.
I think this is more concise and cleaner.